src/blockchain/index.js

```diff
@@ -1,5 +1,5 @@
 const Block = require('./block');
-const DbBlock = require('../model/Block');
+const DbBlock = require('../model/typescript/Block');
```

Comment:
Why do we need a typescript folder? It should be at the root level. Please fix this.